### PR TITLE
feature/OC-438/edit-organisation-tabs

### PR DESCRIPTION
**Pull request by remko48**

### CODE REVIEW

File: pwa/src/templates/templateParts/organizationsForm/OrganizationFormTemplate.module.css [hunk 1]
```diff
@@ -58,3 +58,21 @@
...
+.tabPanel {
+  padding-inline-start: var(--gateway-ui-size-lg) !important;
+  padding-inline-end: var(--gateway-ui-size-lg) !important;
+  padding-block-start: var(--gateway-ui-size-lg) !important;
+  padding-block-end: var(--gateway-ui-size-lg) !important;
+}
+
+.tab {
+  max-width: unset !important;
+}
```
Comment (lencodes) [inline pwa/src/templates/templateParts/organizationsForm/OrganizationFormTemplate.module.css:78]: Out of scope for this pull request, I think, but: we should probably move this to a more generic place.

File: pwa/src/templates/templateParts/organizationsForm/EditOrganizationTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                        <TableCell>No users found</TableCell>
+                        <TableCell />
+                        <TableCell />
+                        <TableCell />
+                      </TableRow>
+                    )}
+                  </TableBody>
+                </Table>
+              </TabPanel>
+              <TabPanel className={styles.tabPanel} value="2   ">
```
Comment (lencodes) [inline pwa/src/templates/templateParts/organizationsForm/EditOrganizationTemplate.tsx:175]: Typo here breaks this tab `"2    "` => `"2"`.

**Review by lencodes — APPROVED** [review]
Looks good. Refactor the one small comment before merging please.

File: pwa/src/templates/templateParts/organizationsForm/OrganizationFormTemplate.module.css (hunk 1 above)
Comment (remko48) [reply pwa/src/templates/templateParts/organizationsForm/OrganizationFormTemplate.module.css:78]: I agree

**Pull request merged**